media/js/xeditable/bootstrap-editable-place.js

```diff
@@ -38,55 +44,54 @@ Place editable input.
     $.fn.editableutils.inherit(Place, $.fn.editabletypes.abstractinput);
 
     $.extend(Place.prototype, {
-        geocodePosition: function() {
-            var self = this;
-            self.geocoder.geocode({
-                latLng: self.marker.getPosition(),
-            }, function(responses) {
-                self.$city.val('');
-                self.$country.val('');
-                self.$address.val('');
-
-                if (responses && responses.length > 0) {
-                    var components = responses[0].address_components;
-                    components.forEach(function(component) {
-                        for (var i=0; i < component.types.length; i++) {
-                            var type = component.types[i];
-                            if (type === 'locality') {
-                                self.$city.val(component.long_name);
-                            }
-                            if (type === 'country') {
-                                self.$country.val(component.long_name);
-                            }
-                        }
-                    });
-                }
-            });
+        markupName: function(obj) {
+            let str = '<div class="separator"><div>' + obj.name;
+            if (obj.adminName1) {
+                str += ', ' + obj.adminName1;
+            }
+            str += '</div><div>' + obj.countryName + '</div>';
```

Comment:
Is there a missing closing `</div>` here? I'm counting three open `<div>` tags, two closing ones.

Reply:
good catch! i've resolved.